src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ListingDetailsSteps.java

```diff
@@ -1,60 +1,71 @@
 package gov.healthit.chpl.aqa.stepDefinitions;
-
 import cucumber.api.java.en.Given;
 import cucumber.api.java.en.Then;
 import cucumber.api.java.en.When;
 import org.apache.commons.lang3.StringUtils;
 import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
+
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertTrue;
-
 import gov.healthit.chpl.aqa.pageObjects.ListingDetailsPage;
 
 /**
- * Class SEDTweaksSteps definition.
+ * Class DeleteDupListingsSteps definition.
  */
-public class SEDTweaksSteps {
 
+public class ListingDetailsSteps {
     private WebDriver driver;
     private static final int TIMEOUT = 30;
     private String url = System.getProperty("url");
 
     /**
      * Constructor creates new driver.
      */
-    public SEDTweaksSteps() {
+    public ListingDetailsSteps() {
         driver = Hooks.getDriver();
         if (StringUtils.isEmpty(url)) {
             url = "http://localhost:3000/";
-        }
+           }
     }
-
     /**
-     * Loads a listing. Waits until the Listing name exists.
-     * @param chplId the database id of the requested listing
+     * Assert that the Accessibility Standard equals the passed in value.
+     * @param accessibilityText expected text
      */
-    @Given("^I am on the Details page of Listing \"([^\"]*)\"$")
-    public void iAmOnAListingsDetailsPage(final String chplId) {
-        driver.get(url + "#/product/" + chplId);
-        WebDriverWait wait = new WebDriverWait(driver, TIMEOUT);
-        wait.until(ExpectedConditions.visibilityOf(ListingDetailsPage.listingName(driver)));
+    @Then("^Accessibility Standard should display \"([^\"]*)\"$")
+    public void accessibilityStandardShouldDisplayUpdatedOtherWCAG(final String accessibilityText) {
+        String actualString = ListingDetailsPage.accessibilityStandardText(driver).getText();
+        assertEquals(actualString, accessibilityText);
     }
-
     /**
-     * Opens the SED section of the accordion.
+     * Loads a listing for given Database ID.
+     * @param dbId the Database Id of a listing to load
      */
-    @When("^I open SED details accordion$")
-    public void iOpenSEDAccordion() {
-        // to close certification criteria accordion that's open by default
-        ListingDetailsPage.certificationCriteriaAccordion(driver).click();
-
-        //to open SED accordion
-        ListingDetailsPage.sedAccordion(driver).click();
+    @Given("^I am on listing details page of listing with database ID \"(.*)\"$")
+    public void loadListingWithDbId(final String dbId) {
+        driver.get(url + "#/product/" + dbId);
+    }
```

Comment:
Definitely want a "wait until" condition here, as the details page takes a long time to load